R/agg.R

```diff
@@ -10,11 +10,15 @@ preprocess <- function(x, q = 0) {
   #' @param x A vector of forecasts
   #' @param q The quantile to use for replacing 0s and 1s (between 0 and 1)
   #' @return A vector of forecasts with 0s are replaced by the qth quantile and
-  #' 100s are replaced by the (1 - q)th quantile.
+  #' 1s are replaced by the (1 - q)th quantile.
   #'
   #' @importFrom stats quantile
   #'
-  #' @note Assumes forecasts are in the range 0 to 100, inclusive.
+  #' @note Assumes forecasts are in the range 0 to 1, inclusive.
+
+  if (any(x > 1)) {
```

Comment:
@hickman-santini should you check `x<0`? or is that impossible (forgive the ignorant comment if it is!)

Reply:
Great idea!! It's an imperfect check anyway (they could be using 0-100 scale and just dealing in <1% probabilities) but it's never a bad idea to check for other weirdness